webclient/package.json

```
@@ -40,7 +41,13 @@
     "eject": "react-scripts eject",
     "lint": "eslint \"./**/*.{ts,tsx}\"",
     "lint:fix": "eslint \"./**/*.{ts,tsx}\" --fix",
-    "golden": "npm run lint && npm run test"
+    "golden": "npm run lint && npm run test",
+    "prepare": "cd .. && husky install webclient/.husky"
```

Does this not work? Also, will this command work on windows?
```suggestion
    "prepare": "husky install ./.husky"
```

---

It might make sense to add this command to the env-specific pre-install copy files.

---

I wanted to have husky in the webclient directory but am happy to move it to parent directory with the pre-push hook

---

husky install has to be ran in the same directory as the .git/ hence the cd. I have moved the .husky up to the parent directory. This is working on windows with powershell